docs/research-software/mitgcm/mitgcm.md

```diff
@@ -99,3 +104,163 @@ each for up to one hour.
     #   Using 256 MPI processes and 128 MPI processes per node
     #   srun picks up the distribution from the sbatch options
     srun --cpu-bind=cores ./mitgcmuv
+    
+## Reproducing the ECCO version 4 (release 4) state estimate on ARCHER2
+
+The ECCO version 4 state estimate (ECCOv4-r4) is an observationally-constrained numerical solution produced by the ECCO group at JPL. If you would like to reproduce the state estimate on ARCHER2 in order to create customised runs and experiments, follow the instructions below. They have been slightly modified from the JPL instructions for ARCHER2. 
+
+For more information, see the ECCOv4-r4 website <https://ecco-group.org/products-ECCO-V4r4.htm>
+
+### Get the ECCOv4-r4 source code
+
+First, create a working directory, perhaps MYECCO. Navigate into this working directory:
```

Comment:
May be worth noting that, on ARCHER2, this must be done in /work to make this available on the compute nodes.

Reply:
Yes, good point. I will add that.